pkgs/development/embedded/arduino/arduino-language-server/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub}:
+
+
+buildGoModule rec {
+  pname = "arduino-language-server";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "arduino";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-FnkXtjktj22LCkmFPxzQ72qiVBC1WR/ZiwMsmZPL578=";
```

Comment:
```suggestion
    hash = "sha256-FnkXtjktj22LCkmFPxzQ72qiVBC1WR/ZiwMsmZPL578=";
```

Reply:
Is this just because sha256 is already specified by the hash?

Reply:
Look at https://github.com/NixOS/nixpkgs/blob/162b38410d85a3135c436e1e9ed44cb5ee6bd549/pkgs/build-support/fetchurl/default.nix#L60-L69

Reply:
Aaah. I see.